package.json

```diff
@@ -47,10 +46,6 @@
     "rwlock": "^5.0.0"
   },
   "devDependencies": {
-    "eslint": "^7.32.0",
```

Comment:
eslint 7 couldn't handle the nullish coalescing operator, and eslint 9 didn't work with eslint-config-node-important-stuff, since eslint-plugin-node hasn't been updated for eslint 9 yet. So I just removed the linting for now.

This whole project needs to be rewritten, due to how unnecessarily complex it is.